frontend/degree-plan/components/Search/CourseInSearch.tsx

```diff
@@ -171,10 +163,12 @@ export default function Course({
             <ReviewPanelTrigger full_code={course.id} triggerType="click">
                 <CourseContainer onMouseEnter={() => setIsMouseOver(true)} onMouseLeave={() => setIsMouseOver(false)}>
                     <CourseInfoContainer
+                        ref={drag} 
+                        className="draggable"
                         role="button"
                     >   
                         <CourseIdentityContainer >
-                            <CourseIDContainer ref={drag} className="draggable">
```

Comment:
do we want to revert this? doesn't this allow the course to be draggable?

Reply:
Oh this is because I put the ref in the parent CourseInfoContainer component, but I think putting it in the CourseIDContainer component makes more sense in terms of UI. I'll change it.

Reply:
Actually I'll put it in CourseIdentityContainer

Reply:
Oh oops missed that it was on the parent. Do whatever makes sense to you!